<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificaRichiesta</class>
 <widget class="QDialog" name="ModificaRichiesta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>259</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>753</width>
    <height>259</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>753</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modifica Richiesta</string>
  </property>
  <widget class="QPushButton" name="BtnMaterialeEsci">
   <property name="geometry">
    <rect>
     <x>642</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Esci</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalva">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifica richiesta</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpDataRichiesta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Richiesta</string>
   </property>
   <widget class="QDateEdit" name="dateEditRichiesta">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpCalendari">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>120</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Calendari</string>
   </property>
   <widget class="QComboBox" name="comboBoxCalendari">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBuste">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Buste</string>
   </property>
   <widget class="QLabel" name="lblRosse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rosse</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBlu">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Blu</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVerdi">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Verdi</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBianche">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bianche</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRosse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 0, 0, 50);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxBlu">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(9, 47, 252, 50);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxVerdi">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(3, 251, 63, 50);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxBianche">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpSecchi360">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Secchi 360 lt</string>
   </property>
   <widget class="QComboBox" name="comboBoxSecchiUmido360">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(193, 94, 7, 50);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxSecchiVetro360">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(106, 144, 126, 50);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblSecchiUmido360">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Umido</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSecchiVetro360">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vetro</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpMastelli">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Mastelli</string>
   </property>
   <widget class="QLabel" name="lblUmidoMastelli">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Umido</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxMastelliUmido">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(193, 94, 7, 50);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblVetroMastelli">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vetro</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxMastelliVetro">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(106, 144, 126, 50);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpSecchi240">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Secchi 240 lt</string>
   </property>
   <widget class="QComboBox" name="comboBoxSecchiUmido240">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(193, 94, 7, 50);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxSecchiVetro240">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(106, 144, 126, 50);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblSecchiUmido240">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Umido</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSecchiVetro240">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vetro</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
